README.md

```diff
@@ -80,9 +80,9 @@ cd ios && pod install
 
 Synchronously starts the Hermes Profiling Session.
 
-### `stopProfiling(saveInDownloadsDirectory: boolean): Promise<string>`
+### `stopProfiling(saveInDownloadsDirectory: boolean, fileName: string): Promise<string>`
```

Comment:
this is an optional, right? Let's add a `?` to the param type here

Reply:
@hannojg actually `saveInDownloadsDirectory` is also optional (false by default), but it's not marked as optional in README. Should we mark both params as optional then?

Reply:
I think we should yes!

Reply:
Done!